editions/tw5.com/tiddlers/filters/search.tid

```diff
@@ -1,21 +1,44 @@
 created: 20140410103123179
-modified: 20150203191048000
+modified: 20181025082022690
 tags: [[Filter Operators]] [[Common Operators]] [[Field Operators]] [[Negatable Operators]]
 title: search Operator
 type: text/vnd.tiddlywiki
 caption: search
 op-purpose: filter the input by searching tiddler content
 op-input: a [[selection of titles|Title Selection]]
-op-suffix: optionally, the name of a [[field|TiddlerFields]]
-op-suffix-name: F
-op-parameter: one or more search terms, separated by spaces
+op-suffix: the <<.op search>> operator uses a rich suffix, see below for details
+op-parameter: one or more search terms, separated by spaces, or a literal search string
 op-output: those input tiddlers in which <<.em all>> of the search terms can be found in the value of field <<.place F>>
 op-neg-output: those input tiddlers in which <<.em not>> all of the search terms can be found
 
-When used with a suffix, the <<.op search>> operator is similar to <<.olink regexp>> but less powerful.
+<<.from-version "5.1.18">> Earlier versions do not support searching multiple fields, or the ''literal'' or ''casesensitive'' options.
```

Comment:
`<<.from-version "5.1.18">> ` usually shows a "new in Version 5.1.18" tag .... but the text talks about "Earlier versions"... So IMO it should be rephrased, to talk about the new possibilities.

Reply:
Thanks @pmario